extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/promptSaveReport.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import { Dialog, Input } from '@ohif/ui';
+import createReportAsync from './../../_shared/createReportAsync.js';
+
+const RESPONSE = {
+  NO_NEVER: -1,
+  CANCEL: 0,
+  CREATE_REPORT: 1,
+  ADD_SERIES: 2,
+  SET_STUDY_AND_SERIES: 3,
+  NO_NOT_FOR_SERIES: 4,
+};
+
+function promptUser({ servicesManager, extensionManager }, ctx, evt) {
+  const { UIDialogService, MeasurementService } = servicesManager.services;
+  const { StudyInstanceUID, SeriesInstanceUID } = evt;
+  const { trackedStudy, trackedSeries } = ctx;
+
+  return new Promise(async function(resolve, reject) {
+    // TODO: Fallback if (UIDialogService) {
+    const promptResult = await _createReportDialogPrompt(UIDialogService);
+
+    if (promptResult.action === RESPONSE.CREATE_REPORT) {
+      // TODO: use `promptResult.value` to set seriesDescription
```

Comment:
`promptResult.value` is the input value at time of dialog form submission. This needs to set the series description for the SR and any SR specific title field

Reply:
👍

Reply:
Delete this TODO comment which is now done.

Reply:
Fixed.